CONTRIBUTORS.md

```diff
@@ -167,7 +167,7 @@ Add your **i) Github username** and **ii) GitHub profile link** to the **Contrib
 157. [syaddadSmiley](https://github.com/syaddadSmiley)
 158. [Pratyushdube](https://github.com/Pratyushdube)
 159. [prince-63](https://github.com/prince-63)
-160. [shashikant62](https://github.com/shashikant62)
+60. [shashikant62](https://github.com/shashikant62)
```

Comment:
@Ayushigupta7453, in your PR, I believe there is an accidental change in the number for line 170.

Please revert to the original value for line 170, and make the necessary changes.

Ping me again once the changes have been made.

Reply:
resolved this issue